lib/util/memcache.ex

```diff
@@ -4,12 +4,51 @@ use Croma
 
 defmodule Antikythera.Memcache do
   @moduledoc """
-  In-memory cache for each executor pool.
+  Easy-to-use in-memory cache for each executor pool.
 
-      iex> Antikythera.Memcache.write("foo", "bar", epool_id)
+  #{inspect(__MODULE__)} behaves as a key-value storage.
+  Cached key-value pairs are internally stored in ETS.
+  It accepts arbitrary (but not too large) terms as both keys and values.
+
+  ## Usage
+
+      iex> Antikythera.Memcache.write("foo", "bar", epool_id, 3_600)
       :ok
       iex> Antikythera.Memcache.read("foo", epool_id)
       {:ok, "bar"}
+
+  ## Limits
+
+  The number of records and the size of keys and values is limited.
```

Comment:
Should be "... are limited."

Reply:
Fixed in 530f5de234e4c4a344ca0e261f82e7c5ece7e0fb